jellyfin-api/src/main/kotlin/org/jellyfin/apiclient/api/client/KtorClient.kt

```diff
@@ -98,12 +98,20 @@ public open class KtorClient(
 		}.buildString()
 	}
 
+	/**
+	 * This is a temporary workaround until we have a proper way to send device names to the server.
+	 * It will filter out all special characters.
+	 */
+	private fun String.encodeAuthorizationHeaderValue() = replace("[^\\w\\s]".toRegex(), "").trim()
```

Comment:
Wouldn't URL-escaping be preferable here? Or doesn't the server like it then?

Reply:
Server hates it :( It will just save it with urlencoding then

Reply:
I see, then we should keep it like this.